data/json/items/vehicle/motors.json

```diff
@@ -76,5 +76,16 @@
     "category": "veh_parts",
     "price": 2000,
     "price_postapoc": 100
+  },
+  {
+    "type": "GENERIC",
+    "id": "motor_train1300",
+    "name": { "str": "1300hp electric train engine" },
+    "description": "A 1300hp 3-phase 60 Hz electrict engine.  Normally used in trains.",
+    "weight": "2000 kg",
+    "volume": "45 L",
```

Comment:
This engine is about twice as dense as osmium like this, so I suspect it may be a bit larger than 45L.

Reply:
does 250 L sound better? it's closer to steel's density now

Reply:
Yep.